scripts/cubecam.lua

```diff
@@ -38,6 +38,19 @@ local zoom_map = {
   4,
 }
 
+local function cubecam_calculate_size(player, fullscreen)
+  local resolution = player.display_resolution
+  if fullscreen then
+    local scale = player.display_scale
+    return {resolution.width / scale, resolution.height / scale}
+  else
+    local scale_settings = player.mod_settings["cube-cubecam-scale"].value / 100.0
+    local inverse_display_scale = math.abs(2.0 - player.display_scale)
```

Comment:
not sure about this line, if you're trying to make the cubecam take up the same amount of screen real estate regardless of display scaling, don't you just want to divide by `display_scale`? This seems like it'll always size the window to zero for a display scale of 200%. (But it might make more sense to just let the window adjust with the display scale like everything else)